### PR TITLE
UD-913 avoid blinking effect of list empty state

### PR DESCRIPTION
**Pull request by akurinnoy**

### CODE REVIEW

File: src/app/projects/list-projects/list-projects.html [hunk 1]
```diff
@@ -40,11 +40,11 @@
     class="projects-list-projects-filter-action-icon material-design icon-ic_close_24px"></i></div>
 </div>
 
-<md-content md-scroll-y flex layout="column" layout-align="center" class="projects-list-projects" md-theme="maincontent-theme">
```
Comment (benoitf) [inline src/app/projects/list-projects/list-projects.html]: why this layout change is affecting the blink ?

File: src/app/workspaces/list-workspaces/list-workspaces.html [hunk 2]
```diff
@@ -20,10 +20,10 @@
   <span flex></span>
 </md-toolbar>
 
-<md-content md-scroll-y flex layout="column" layout-align="center" md-theme="maincontent-theme">
```
Comment (benoitf) [inline src/app/workspaces/list-workspaces/list-workspaces.html]: same question there ?

Comment (akurinnoy) [inline src/app/workspaces/list-workspaces/list-workspaces.html]: md-linear-progress then is aligned at center

File: src/app/projects/list-projects/list-projects.html (hunk 1 above)
Comment (akurinnoy) [inline src/app/projects/list-projects/list-projects.html]: md-linear-progress then is aligned at center

**Comment by benoitf** [comment]
ok I see. Thx
+1 for me

**Pull request merged**